### PR TITLE
Convert jsonbuilder/v1

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) tickets:
https://redmine.openinfosecfoundation.org/issues/3756
https://redmine.openinfosecfoundation.org/issues/3754

### CODE REVIEW

File: rust/src/krb/log.rs [hunk 1]
```diff
@@ -48,11 +46,19 @@ pub extern "C" fn rs_krb5_log_json_response(_state: &mut KRB5State, tx: &mut KRB
...
-    js.set_string("encryption", &encryption);
-    js.set_boolean("weak_encryption", tx.etype.map_or(false,test_weak_encryption));
-    return js.unwrap();
+    jsb.set_string("cname", &cname)?;
+    jsb.set_string("realm", &realm)?;
+    jsb.set_string("sname", &sname)?;
+    jsb.set_string("encryption", &encryption)?;
+    jsb.set_bool("weak_encryption", tx.etype.map_or(false,test_weak_encryption))?;
+
+    jsb.close()?;
```
Comment (victorjulien) [inline rust/src/krb/log.rs:55]: what is closed here?

File: rust/src/snmp/log.rs [hunk 2]
```diff
@@ -37,48 +37,54 @@ fn str_of_pdu_type(t:&PduType) -> Cow<str> {
...
             _           => ()
         }
         match tx.usm {
-            Some(ref s) => js.set_string("usm", s),
+            Some(ref s) => {jsb.set_string("usm", s)?;},
             _           => ()
         }
     }
-    js.unwrap()
+    jsb.close()?;
```
Comment (victorjulien) [inline rust/src/snmp/log.rs:81]: where is the matching open?

**Comment by inashivb** [comment]
> I fear both have a close too many, can you double check?
> 
> Do we have SV tests that validate the output?

I didn't write any specially for this but the existing ones passed. Should there be more?

File: rust/src/snmp/log.rs (hunk 2 above)
Comment (inashivb) [reply rust/src/snmp/log.rs:81]: There isn't. I thought I would get a panic for too many closes. So, didn't pay much attention. Fixing.

**Comment by victorjulien** [comment]
The failure will only lead to panic with `--enable-debug-validation` enabled.

**Comment by inashivb** [comment]
Replaced with #5171

**Pull request closed without merging**